src/ssl.c

```diff
@@ -43056,7 +43056,7 @@ int wolfSSL_RSA_public_decrypt(int flen, const unsigned char* from,
 
     if (rsa == NULL || rsa->internal == NULL || from == NULL) {
         WOLFSSL_MSG("Bad function arguments");
-        return WOLFSSL_FAILURE;
+        return -1;
```

Comment:
Use `WOLFSSL_FATAL_ERROR` instead of `-1` in this function.

Reply:
Ok, fixed.